x-pack/plugins/observability/public/utils/get_alert_evaluation_unit_type_by_rule_type_id.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+export enum AlertEvaluationUnitType {
```

Comment:
TS enums should be avoided: https://www.youtube.com/watch?v=jjMbPt_H3RQ.

Alternative implementation would be a plain old javascript object using `as const`.

Reply:
@CoenWarmer, thanks. I will update it.